ProjectCoimbra.UWP/Project.Coimbra/Project.Coimbra.csproj

```diff
@@ -94,8 +94,36 @@
     </AppxManifest>
   </ItemGroup>
   <ItemGroup>
-    <Content Include="Assets\logo-splash.png" />
-    <Content Include="Assets\logo.png" />
+    <Content Include="Assets\logo-splash.scale-100.png" />

```

Comment:
It would have been nice if we could just have one size of image and then resize using a build pipeline task. That's going to be a bit difficult though, so definitely out of scope for now -- and I'm not sure how we can do it with GitHub. But it would make this more maintainable.

Reply:
indeed. However Visual Studio has an auto generate in the app package viewer that was used to generate the images here, so it should work until we can automate these using build pipelines.